charts/brigade/values.yaml

```diff
@@ -59,6 +71,17 @@ kashti:
 brigade-github-app:
   enabled: false
   github:
+    affinity: {}
```

Comment:
See note elsewhere re: preference to move the `affinity` section up a level (so, immediately under `brigade-github-app`)